docs/user/openstack/privileges.md

```diff
@@ -0,0 +1,49 @@
+# Required Privileges
+
+In order to succesfully deploy an OpenShift cluster on OpenStack, the user passed to the installer needs a particular set of permissions in a given project. We believe that the average user with a member role in a project will already have most, if not all, of the privileges enumerated here. However, due to how much variance there is in how OpenStack clusters are deployed and managed, we designed this document as a guide for the basic permissions needed for an install based on your use case.
+
+## Default Install Privileges
+
+To install a basic, externally routable, IPI OpenShift cluster using only whats offered in the installer prompts, your user needs this minimum subset of permissions.
```

Comment:
I still think the below priviledges are wrong, you don't need all the permissions for ports, router, subnets,... just the "default" ones in most of the cases. You for instance don't need permissions to modify/delete ports in you subnet whose device onwer is router_interface or things like that. So the next information is misleading